proto/meta.proto

```diff
@@ -482,6 +485,20 @@ service NotificationService {
   rpc Subscribe(SubscribeRequest) returns (stream SubscribeResponse);
 }
 
+message BackfillResponse {
+   // holds all MVs which have completed their backfill operations
+   // contains IDs from catalog.Table.id
+   repeated uint32 mv_backfills_completed = 30;
+}
+
+message BackfillRequest {
+   string requestId = 1; // UUID so meta can drop duplicate requests 
+}
+
+service BackfillService {
+   rpc GetBackfill(BackfillRequest) returns (BackfillResponse);
+}
```

Comment:
@shanicky Are you still ok with this? If not, then we could also add a flag `mv_completed` in the `catalog.Table` message.